### ISSUE TITLE
`solid/reactivity` rule triggered in `createSignal`

### ISSUE THREAD

**Issue by baptisteArno** [solidjs-community/eslint-plugin-solid#53]
**Describe the bug**
```
const [inputValue, setInputValue] = createSignal(
    props.block.prefilledValue ?? ''
  )
```

`solid/reactivity` is triggered on this code, but it shouldn't IMO? Is it possible to detect when we are in a `createSignal`?

**Comment by joshwilsonvu** [comment]
`solid/reactivity` exists to help catch cases where you might expect Solid's reactivity system to update when your state changes but the update will be ignored. This is one of those cases--the signal won't react to a change in the prop you're using to initialize it--but that's a common use case and it's okay.

The rule will stop warning if you use a prop that starts with `initial` or `default`, like `props.defaultValue`. Since that doesn't seem feasible in your case, just ignore it with a `// eslint-disable-next-line solid/reactivity` comment!

Unless I get much more pushback I do still want to keep the warning in this case, for consistency. Hopefully this wasn't too much of an inconvenience 🙂
